client/post-editor/editor-ground-control/index.jsx

```diff
@@ -16,7 +16,7 @@ import { connect } from 'react-redux';
  */
 import Card from 'components/card';
 import Site from 'blocks/site';
-import * as postUtils from 'lib/posts/utils';
+import { isPage, isPublished } from 'lib/posts/utils';
```

Comment:
little known fact: webpack is just as good at tree shaking a `*` import as explicit named imports.

Reply:
Good to know 👍